homeassistant/components/tuya/climate.py

```diff
@@ -163,15 +163,15 @@ def __init__(
         celsius_type = self.find_dpcode(
             (DPCode.TEMP_CURRENT, DPCode.UPPER_TEMP), dptype=DPType.INTEGER
         )
-        farhenheit_type = self.find_dpcode(
+        fahrenheit_type = self.find_dpcode(
```

Comment:
Unrelated changes, should normally not be in feature PRs and split into their own PRs.